Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
+ENV PATH $PATH:/usr/local/bin
+ENV TERRAFORM_VER 0.6.7
+ENV TERRAFORM_ZIP terraform_"$TERRAFORM_VER"_linux_amd64.zip
+
+RUN apt-get update && apt-get -y install wget unzip openssh-client dnsutils git pass ruby
```

Comment:
CF is out of scope for now - we're only provisioning Concourse to start with. Either way we should keep the container as minimal as possible and pass (no pun intended!) the credentials as an input.

Thanks for the list. Could you add it to the commit message for posterity?

Similarly do we need dnsutils? If we were debugging in a container then we can add those in at runtime.